src/devices/tuya.ts

```diff
@@ -565,6 +565,7 @@ const definitions: Definition[] = [
             tuya.whitelabel('TuYa', 'MC500A', 'Door sensor', ['_TZ3000_2mbfxlzr']),
             tuya.whitelabel('TuYa', '19DZT', 'Door sensor', ['_TZ3000_n2egfsli']),
             tuya.whitelabel('TuYa', 'DS04', 'Door sensor', ['_TZ3000_yfekcy3n']),
+            tuya.whitelabel('Moes', 'Smart Door and Window Sensor', 'Door sensor', ['_TZ3000_decxrtwa']),
```

Comment:
Please use a proper model number he, something that is written on the device/box.

Reply:
Sorry, my fault... added.